core/src/components/picker-column/picker-column.tsx

```diff
@@ -356,9 +379,20 @@ export class PickerColumnCmp implements ComponentInterface {
     const selectedIndex = clamp(min, this.col.selectedIndex ?? 0, max);
     if (this.col.prevSelected !== selectedIndex || forceRefresh) {
       const y = selectedIndex * this.optHeight * -1;
+      const duration = animated === false ? 0 : TRANSITION_DURATION;
```

Comment:
```suggestion
      const duration = animated ? TRANSITION_DURATION : 0;
```

We try to write positive conditions so they are easy to understand in the event we need to negate them. For example "not animated" is easier to understand than "not not animated".

This one is my bad since I gave you this code snippet, and I should have written it in the positive to begin with.